test/integration/ferc1_eia_train_test.py

```diff
@@ -0,0 +1,292 @@
+            ["f1_steam_2020_12_161_0_1"],  # FERC1 id NOT in training data
+            marks=pytest.mark.xfail(reason="EIA id already in training data."),
+        ),
+        pytest.param(
+            ["True"],
+            [2020],
+            # NOTE: These are not matches, just test examples
+            ["1_2020_plant_owned_63560"],  # EIA id NOT in training data
+            ["f1_steam_2020_12_134_3_1"],  # FERC1 id already in training data
+            marks=pytest.mark.xfail(reason="FERC1 id already in training data."),
+        ),
+    ],
+)
+def test_validate_override_fixes_no_overrides(
+    utils_eia860,
+    plant_parts_eia,
+    ferc1_eia,
+    ferc1_eia_training_data,
+    verified,
+    report_year,
+    record_id_eia_override_1,
+    record_id_ferc1,
+):
+    """Test validate override fixes function where expect_override_overrides=False."""
+    test_df = pd.DataFrame(
+        {
+            "verified": verified,
+            "report_year": report_year,
+            "record_id_eia_override_1": record_id_eia_override_1,
+            "record_id_ferc1": record_id_ferc1,
+        }
+    ).assign(verified=lambda x: x.verified.astype("bool"))
+    ferc1_eia_training_data_restricted = restrict_train_connections_on_date_range(
+        train_df=ferc1_eia_training_data,
+        id_col="record_id_ferc1",
+        start_date=min(ferc1_eia.report_date),
+        end_date=max(ferc1_eia.report_date),
+    )
+    validate_override_fixes(
+        validated_connections=test_df,
+        utils_eia860=utils_eia860,
+        ppe=plant_parts_eia,
+        ferc1_eia=ferc1_eia,
+        training_data=ferc1_eia_training_data_restricted,
+        expect_override_overrides=False,
+        allow_mismatched_utilities=True,
+    )
+
+
+@pytest.mark.parametrize(
+    "verified,report_year,record_id_eia_override_1,record_id_ferc1,utility_id_eia,utility_id_pudl",
+    [
+        pytest.param(
+            ["True"],
+            [2020],
+            # NOTE: These are not matches, just test examples
+            ["1_2020_plant_owned_63560"],
+            ["f1_steam_2020_12_161_0_1"],
+            [63560],
+            [1],
+            marks=pytest.mark.xfail(reason="Utilities don't match"),
+        ),
+        pytest.param(
+            ["True"],
+            [2020],
+            ["8055_2020_plant_total_814"],
+            ["f1_steam_2020_12_8_0_5"],
+            [814],
+            [106],
+        ),
+    ],
+)
+def test_validate_override_fixes_no_mismatched_utilities(
+    utils_eia860,
+    plant_parts_eia,
+    ferc1_eia,
+    ferc1_eia_training_data,
+    verified,
+    report_year,
+    record_id_eia_override_1,
+    record_id_ferc1,
+    utility_id_eia,
+    utility_id_pudl,
+):
+    """Test validate override fixes function where mismatched_utilities=False.
+
+    The parameters have a field for utility_id_eia and utility_id_pudl because the
+    override spreadsheets also have that field. As written, the
+    """
```

Comment:
this trails off and i am a little confused... why are the utility id's here? don't they normally come from the ppe instead of the spreadsheet?

Reply:
if by the off chance they don't need to be here, it seems like it would be simpler to put all these examples into the first mega test and add the `expect_override_overrides` and `allow_mismatched_utilities` flags to the params.